serverless/README.md

```diff
@@ -121,7 +121,8 @@ To further configure your plugin, use the following custom parameters:
 | `nodeLayerVersion`          | Version of the Node.js Lambda layer to install, such as "29". Required if you are deploying at least one Lambda function written in Node.js and `addLayers` is true. Find the latest version number from [https://github.com/DataDog/datadog-lambda-js/releases][6].                                                                                                                                                                                                                                |
 | `dotnetLayerVersion`        | Version of the .NET Lambda layer to install, such as "14". Required if you are deploying at least one Lambda function written in .NET and `addLayers` is true. Find the latest version number from [https://github.com/DataDog/dd-trace-dotnet-aws-lambda-layer/releases][9].
 | `javaLayerVersion`        | Version of the Java Lambda layer to install, such as "12". Required if you are deploying at least one Lambda function written in Java and `addLayers` is true. Find the latest version number from [https://github.com/DataDog/datadog-lambda-java/releases][10].
-| `extensionLayerVersion`     | Version of the Datadog Lambda Extension layer to install, such as "5". When `extensionLayerVersion` is set, `apiKey` (or if encrypted, `apiKMSKey` or `apiKeySecretArn`) needs to be set as well. While using `extensionLayerVersion` do not set `forwarderArn`. Learn more about the Lambda extension [here][8].                                                                                                                                                                                   |
+| `addExtension`                 | Whether to add the Lambda extension layer to the Lambda function. Defaults to true. When true, the extension layer is applied which handles sending logs, traces and custom metrics. The Parameter `extensionLayerVersion` must be set. While using `extensionLayerVersion` do not set `forwarderArn`.  Learn more about the Lambda extension [here][8].                                                                                                                                                                                                                                        |
```

Comment:
Non-blocking, but if you need the `extensionLayerVersion` to be set first, recommend moving the `addExtension` to after the `extensionLayerVersion`.

```suggestion
| `addExtension`                 | Whether to add the Lambda extension layer to the Lambda function. Defaults to true. When true, the extension layer is applied which handles sending logs, traces, and custom metrics. The Parameter `extensionLayerVersion` must be set. While using `extensionLayerVersion` do not set `forwarderArn`.  Learn more about the Lambda extension [here][8].                                                                                                                                                                                                                                        |
```

Reply:
@estherk15 -- For future reference is their a documentation best practices guide?
I was debating the order of the these two parameters. I went with order of appearance then had the reordered the related parameters right beside it.

Also should I reorder the the `addLayers` parameter and the version number parameters for each runtime?

Reply:
Thanks for the context @TophrC-dd, if a user is coming to this page as a reference to learn more about these parameters, I think it's fine as it is. It's best practice start with requirements, but I think in this case, it's not as applicable.